fishtest/fishtest/views.py

```diff
@@ -339,58 +384,63 @@ def parse_spsa_params(raw, spsa):
 
   return params
 
+
 def validate_form(request):
   data = {
-    'base_tag' : request.POST['base-branch'],
-    'new_tag' : request.POST['test-branch'],
-    'tc' : request.POST['tc'],
-    'book' : request.POST['book'],
-    'book_depth' : request.POST['book-depth'],
-    'base_signature' : request.POST['base-signature'],
-    'new_signature' : request.POST['test-signature'],
-    'base_options' : request.POST['base-options'],
-    'new_options' : request.POST['new-options'],
-    'username' : authenticated_userid(request),
-    'tests_repo' : request.POST['tests-repo'],
-    'info' : request.POST['run-info']
+    'base_tag': request.POST['base-branch'],
+    'new_tag': request.POST['test-branch'],
+    'tc': request.POST['tc'],
+    'book': request.POST['book'],
+    'book_depth': request.POST['book-depth'],
+    'base_signature': request.POST['base-signature'],
+    'new_signature': request.POST['test-signature'],
+    'base_options': request.POST['base-options'],
+    'new_options': request.POST['new-options'],
+    'username': authenticated_userid(request),
+    'tests_repo': request.POST['tests-repo'],
+    'info': request.POST['run-info']
```

Comment:
If I recall correctly in another PR you added a trailing comma after the latest element because the closing )]} was on the following line:
```
    'info': request.POST['run-info'],
  }
```

Also suggested by http://google.github.io/styleguide/pyguide.html#341-trailing-commas-in-sequences-of-items